types/wordpress__customize/wordpress__customize-tests.ts

```diff
@@ -0,0 +1,49 @@
+import * as _ from 'underscore';
+
+declare const wp: {
+	customize: typeof import('wordpress__customize');
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/wordpress__customize/wordpress__customize-tests.ts;
  Line: 4;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.